Styles/style.css

```diff
@@ -0,0 +1,244 @@
+/* DELETE LATER
+.border-red {
+  border: red 2px solid;
+}
+.border-green {
+  border: green 2px solid;
+}
+.border-blue {
+  border: blue 2px solid;
+} */
+
+/* COLOR */
+html {
+  --primary-color: rgb(255, 149, 149);
+  --second-primary-color: rgb(252, 85, 112);
+  --accent-color: rgb(214, 213, 213);
+  --second-accent-color: rgb(104, 99, 99);
+}
```

Comment:
Well done for organising your colours this way. It's advanced, neat and you can change a colour easily. I need to implement this method more often to my own projects.